client-libraries/rust/modality-ingest-client/src/types.rs

```diff
@@ -300,3 +373,102 @@ impl From<bool> for AttrVal {
         AttrVal::Bool(b)
     }
 }
+
+impl From<Nanoseconds> for AttrVal {
+    fn from(ns: Nanoseconds) -> AttrVal {
+        AttrVal::Timestamp(ns)
+    }
+}
+
+impl From<LogicalTime> for AttrVal {
+    fn from(lt: LogicalTime) -> AttrVal {
+        AttrVal::LogicalTime(lt)
+    }
+}
+
+// TODO: Should I actually assume that any UUID is a timeline id?
+impl From<Uuid> for AttrVal {
+    fn from(u: Uuid) -> AttrVal {
+        AttrVal::TimelineId(Box::new(u.into()))
+    }
+}
```

Comment:
Should I automatically assume this? This probably depends on the answer to how we parse strings.

Reply:
Seems fine for now.